presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/TestPostgreSqlTypeMapping.java

```diff
@@ -1336,7 +1360,7 @@ private Session withUnsupportedType(UnsupportedTypeHandling unsupportedTypeHandl
                 TIMESTAMP_WITH_TIME_ZONE,
                 DateTimeFormatter.ofPattern("'TIMESTAMP '''yyyy-MM-dd HH:mm:ss.SSS VV''")::format,
                 // PostgreSQL does not store zone, only the point in time
-                zonedDateTime -> zonedDateTime.withZoneSameInstant(ZoneId.of("UTC")));
+                zonedDateTime -> zonedDateTime.truncatedTo(MILLIS).withZoneSameInstant(ZoneId.of("UTC")));
```

Comment:
why truncate here?

and why only millis, where you're adding precision up to micro?

(if this is needed, a code comment is due)

Reply:
not valid